src/Microsoft.Azure.ServiceBus/Primitives/SharedAccessSignatureToken.cs

```diff
@@ -132,7 +115,7 @@ static string GetAudienceFromToken(string token)
             return audience;
         }
 
-        static void GetExpirationDateAndAudienceFromToken(string token, out DateTime expiresOn, out string audience)
+        static DateTime GetExpirationDateFromToken(string token)
```

Comment:
Is this a UTC time? If so, should the method name indicate so?

Reply:
Yes UTC. Fixed API name.

---
In reply to: [156569166](https://github.com/Azure/azure-service-bus-dotnet/pull/385#discussion_r156569166) [](ancestors = 156569166)